app/models/manageiq/providers/hawkular/middleware_manager/event_catcher/runner.rb

```diff
@@ -67,26 +64,32 @@ def event_monitor_handle
   end
 
   def whitelist?(event)
-    return false unless @whitelist.key?(event.category)
-    pattern = @whitelist[event.category]
-    return true unless pattern
-    message = event.context['Message']
-    message && message.match(pattern)
+    tags = event.tags
+    return false unless tags
+    event_type = tags[TAG_EVENT_TYPE]
+    return false unless event_type && @whitelist.include?(event_type)
+    # For a whitelisted event, add some expected fields
```

Comment:
this adds another concern to this method. 
and imho a crictical, as it modifies the parameter.
how about moving this logic to `event_to_hash` ?

I also think `whitelist?` and `event_to_hash` are good candidates for specs :smile: